static/app/views/replays/detail/networkList.tsx

```diff
@@ -0,0 +1,80 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {PanelTable} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {showPlayerTime} from 'sentry/components/replays/utils';
+import {t} from 'sentry/locale';
+import {EventTransaction} from 'sentry/types';
+
+type NetworkListProps = {
```

Comment:
Nit: It's the only props on the page, so we can just call it props

```suggestion
type Props = {
```